### ISSUE TITLE
CompletionProposal window stay while moving the form.

### ISSUE THREAD

**Issue by NilsHoyer** [bonecode/TBCEditor#563]
I'm trying to implemnt a CompletionProposal window. It works fine, if I open it:
![start](https://cloud.githubusercontent.com/assets/1734822/22710427/8c414206-ed7d-11e6-96cf-d3ac655eda37.png)

Than I move my Form ... but the CompletionProposal stays on the same position:
![error](https://cloud.githubusercontent.com/assets/1734822/22710444/9e4b840c-ed7d-11e6-8289-de26ccad46e1.png)

Maybe you can fix this with the message WM_WINDOWPOSCHANGING:

```Delphi
procedure TWWindow.WMWindowPosChanging(var Message: TWMWindowPosChanging);
begin
  if (Message.WindowPos^.flags and SWP_NOMOVE = 0) then
    HidePopupChildren();

  inherited;
end;
```

**Comment by NilsHoyer** [comment]
Sorry, my mistake: You can't handle the `WM_WINDOWPOSCHANGING`, this it will be sent to the form - not to your popup. If I'm right (now), you can't handle this problem - I have to add this code, to my TForm window, where I placed the TBCEditor in it. :-(

**Comment by bonecode** [comment]
Yes, I noticed that pretty quickly. Only message that comes when moving the form is WM_IME_NOTIFY...
